CODEOWNERS

```diff
@@ -0,0 +1,57 @@
+# This file controls automatic PR reviewer assignment. See the following docs:
+#
+# * https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+# * https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team
+#
+# The goal of this file is for most PRs to automatically and fairly have one
+# maintainer and two reviewers set as PR reviewers. All maintainers have
+# permission to approve and merge PRs, but reviewers do not. Most PRs should by
+# members of the reviewers group before being passed to a maintainer for final
+# review.
+#
+# This in part depends on how the groups in this file are configured.
+#
+# @crossplane/steering-committee - Assigns 3 members. Admin perms to this repo.
+# @crossplane/crossplane-maintainers - Assigns 1 member. Maintain perms to this repo.
+# @crossplane-crossplane-reviewers - Assigns 2 members. Triage perms to this repo.
+#
+# Where possible, prefer explicitly specifying a maintainer who is a subject
+# matter expert for a particular part of the codebase rather than using the
+# @crossplane/crossplane-maintainers group.
+#
+# See also OWNERS.md for governance details
+
+# Fallback owners
+*                                   @crossplane/crossplane-maintainers @crossplane/crossplane-reviewers
```

Comment:
That's a shame. Apparently triage isn't enough. It seems like I would have to grant @crossplane/crossplane-reviewers write access, which I believe would allow them to approve and merge PRs without approval from a member of @crossplane/crossplane-maintainers .

Reply:
<img width="755" alt="Screen Shot 2022-08-12 at 7 31 39 PM" src="https://user-images.githubusercontent.com/1049349/184465229-f2a27bb3-f47b-4107-80bb-9ff492af7c17.png">

Hrm, it could be that our branch protection would prevent folks without "maintain" or higher permissions _merging_ a PR?

Reply:
https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization indicates that write access would typically give folks the ability to merge PRs.

I'll switch the Crossplane-reviewers team over to 'write' on Monday and ask a volunteer in that team to let me know what permissions they have. I'd personally feel comfortable with @crossplane/crossplane-reviewers having permission to approve PRs as long as only @crossplane/crossplane-maintainers can merge them.

Reply:
I've given @crossplane/crossplane-reviewers write permissions to test this.

Reply:
> That's a shame.

not sure if i caught what exactly the shame was here - was there a message or github status or something this was referring to? i may have picked up the context from the rest of your thread though...

yes, our branch protection looks like folks cannot merge PRs without being at least in the maintainer role (write access would not be enough to merge a PR).

the general goal of crossplane-reviewers having permission to APPROVE, but then crossplane-maintainers can MERGE them sounds exactly what we want.

Reply:
> not sure if i caught what exactly the shame was here - was there a message or github status or something this was referring to?

Yeah, it seems like GitHub has a built in CODEOWNERS linter that was telling me that @crossplane/crossplane-reviewers had insufficient permissions to be a code owner. That went away automatically when I bumped them from triage to write.